### PR TITLE
Add exception for java APIs.

### PR DESCRIPTION
**Pull request by xiangpingzhang**
"add exception in java sdk"

Signed-off-by: zhangxiangping <240133996@qq.com>

**What this PR does / why we need it**:

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
```

### CODE REVIEW

**Review by mervinkid — APPROVED** [review]
LGTM

File: java/src/main/java/io/chubao/exception/IOException.java [hunk 1]
```diff
@@ -0,0 +1,10 @@
+package io.chubao.exception;
+
+
+import static io.chubao.exception.StatusCodes.CFS_STATUS_IO;
+
+public class IOException extends CfsException {
```
Comment (mervinkid) [inline java/src/main/java/io/chubao/exception/IOException.java:6]: Why not directly use the built-in `java.io.IOException` as IO related error exception.

Comment (xiangpingzhang) [reply java/src/main/java/io/chubao/exception/IOException.java:6]: You are right, I will modify it right away.

**Review by mervinkid — APPROVED** [review]
LGTM

**Review by shuoranliu — APPROVED** [review]

**Pull request merged**